cloudweatherreport/model.py

```diff
@@ -544,7 +544,7 @@ def from_bundletester_output(cls, provider, output):
                 result_code = 'FAIL'
             result.tests.append(TestResult(
                 name=test['test'],
-                duration=test['duration'],
+                duration=test.get('duration', 0.0),
```

Comment:
- Trace when `duration` was missing https://pastebin.canonical.com/168422/
- It is 0.0 because ScalarFiled in model.py expects float.